dev_tools/conftest.py

```diff
@@ -29,24 +29,37 @@
 def pytest_configure(config):
     config.addinivalue_line("markers", "slow: mark tests as slow")
     config.addinivalue_line("markers", "weekly: mark tests as run only by weekly automation")
+    config.addinivalue_line("markers", "rigetti_integration: tests that connect to Quil compiler or QVM.")
 
 
 def pytest_collection_modifyitems(config, items):
     markexpr = config.option.markexpr
     if markexpr:
         return  # let pytest handle this
 
-    skip_slow_marker = pytest.mark.skip(reason='slow marker not selected')
+    # do not skip slow tests if --enable-slow-tests is passed
```

Comment:
Typically conftest.py in subdirectories should have configurations relevant for that directory tree only.
The intent here is to have extra pytest options and test selection logic at one place.
As these are global settings it is better to move this to the top-level conftest.py.